articles/service-fabric/service-fabric-cicd-your-linux-java-application-with-jenkins.md

```diff
@@ -34,29 +34,33 @@ You can set up Jenkins either inside or outside a Service Fabric cluster. The fo
   ```sh
   sudo apt-get install wget
   wget -qO- https://get.docker.io/ | sh
-  ```
-2. Have the Service Fabric container application deployed on the cluster, by using the following steps:
+  ``` 
+
+   > [!NOTE]
+   > Ensure that the 8081 port is specified as a custom endpoint on the cluster.
+   >
+2. Clone the application, by using the following steps:
 
   ```sh
 git clone https://github.com/Azure-Samples/service-fabric-java-getting-started.git
 cd service-fabric-java-getting-started/Services/JenkinsDocker/
 ```
 
-3. You need the connect option details of the Azure storage file-share, where you want to persist the state of the Jenkins container instance. If you are using the Microsoft Azure portal for the same, please follow the steps - Create an Azure storage account, say ``sfjenkinsstorage1``. Create a **File Share** under that storage account, say ``sfjenkins``. Click on **Connect** for the file-share and note the values it displays under **Connecting from Linux**, say this would look like as follows -
+3. Persist the state of the Jenkins container in a file-share:
+  * Create an Azure storage account in the **same region** as your cluster with a name such as ``sfjenkinsstorage1``.
+  * Create a **File Share** under the storage Account with a name such as ``sfjenkins``.
+  * Click on **Connect** for the file-share and note the values it displays under **Connecting from Linux**, the value should look
+  similar to the one below:
 ```sh
 sudo mount -t cifs //sfjenkinsstorage1.file.core.windows.net/sfjenkins [mount point] -o vers=3.0,username=sfjenkinsstorage1,password=<storage_key>,dir_mode=0777,file_mode=0777
 ```
 
-> [!NOTE]
-> To mount cifs shares, you need to have the cifs-utils package installed in the cluster nodes. 
->
-
-4. Update the placeholder values in the ```setupentrypoint.sh``` script with corresponding azure-storage details.
+4. Update the placeholder values in the ```setupentrypoint.sh``` script with the azure-storage details from step 3.
 ```sh
 vi JenkinsSF/JenkinsOnSF/Code/setupentrypoint.sh
 ```
-Replace ``[REMOTE_FILE_SHARE_LOCATION]`` with the value ``//sfjenkinsstorage1.file.core.windows.net/sfjenkins`` from the output of the connect in point 3 above.
-Replace ``[FILE_SHARE_CONNECT_OPTIONS_STRING]`` with the value ``vers=3.0,username=sfjenkinsstorage1,password=GB2NPUCQY9LDGeG9Bci5dJV91T6SrA7OxrYBUsFHyueR62viMrC6NIzyQLCKNz0o7pepGfGY+vTa9gxzEtfZHw==,dir_mode=0777,file_mode=0777`` from point 3 above.
+Replace ``[REMOTE_FILE_SHARE_LOCATION]`` with the value ``//sfjenkinsstorage1.file.core.windows.net/sfjenkins`` from the output of the connect in step 3 above.
```

Comment:
Since there is no preview link in the PR, wondering how does these 2 'Replace' steps show up in the actual doc? Numbered list or bulletted list?

Reply:
It does not show up in a list at all, it is a paragraph. This is the way it was previously in the document.

Reply:
Okay. Given you are anyway updating it, would you mind making these 2 points a bulletted list.